dev/com.ibm.ws.security.social/test/com/ibm/ws/security/social/internal/utils/OpenShiftUserApiUtilsTest.java

```diff
@@ -229,41 +406,54 @@ public void missingUserKeyFromResponse() {
             });
             String errorResponse = "{\"kind\":\"TokenReview\",\"apiVersion\":\"authentication.k8s.io/v1\",\"metadata\":{\"creationTimestamp\":null},\"spec\":{\"token\":\"somebadvalueForAnAccessToken\"},\"status\":{\"user\":{},\"error\":\"[invalid bearer token, token lookup failed]\"}}";
             userApiUtils.modifyExistingResponseToJSON(errorResponse);
-            //assertEquals(returnedString, "{\"username\":\"admin\",\"groups\":}");
-
         } catch (SocialLoginException e) {
-            //nls 
-            verifyException(e, "CWWKS5374E");
-            // fail()
+            verifyException(e, "KUBERNETES_USER_API_RESPONSE_ERROR");
```

Comment:
I would think this should be looking for `CWWKS5380E` instead of `"KUBERNETES_USER_API_RESPONSE_ERROR"`.